WeakAuras/Prototypes.lua

```diff
@@ -10170,15 +10169,15 @@ Private.event_prototypes = {
       },
       {
         name = "instance_size",
-        display = L["Instance Size Type"],
+        display = L["Instance Size Type"].." "..L["|cffff0000deprecated|r"],
```

Comment:
I think adding a desc that directs to the location trigger here is needed, if we don't add an AuraWarning.